apps/api-server/src/infrastructure/repositories/artistHandleHistoryRepository/index.test.ts

```
@@ -0,0 +1,47 @@
+  changedByUserId: "user-1",
+};
+
+describe("createArtistHandleHistoryWriter", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it("生成時に渡した executor で全フィールドを insert する", async () => {
+    const executor = createExecutor();
+
+    await createArtistHandleHistoryWriter(executor as never).record(data);
```

ESLint

[error] `as` による型アサーションは使用しない。型ガード関数に切り出すか、正しく型付けされた契約から値を取得する。

(no-restricted-syntax)